src/components/InputForm.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { useNote } from "../contexts/InputContext"
+import "./common.css"
+
+export default function InputForm() {
+    const { setNotes } = useNote()
+    const [title, setTitle] = useState("")
+    const [description, setDescription] = useState("")
```

Comment:
Sugestion : How i see this is if you can further have maybe some more additional field say category or you might do some color coding for importance. So in that case if you have a common reducer function that might be helpful too. Since the state might increase from 2 to 3 or more.